capitalizeAndFilter.js

```diff
@@ -0,0 +1,11 @@
+const capitalizeAndFilter = (array) => {
+  const newArray = [];
+  array.filter(string => {
+    if (string.slice(0, 1).toUpperCase() !== 'F'){
```

Comment:
You can split this up to be two separate steps - first capitalize the first letter of each string in the array, then in the second step(in your if statement), filter for F.